src/patternfly/components/AlertGroup/examples/alert-group-example.hbs

```diff
@@ -0,0 +1,54 @@
+{{#> alert-group}}
+  {{#> alert-item}}
+    {{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+      {{#> alert-icon alert-icon--success="true"}}
+      {{/alert-icon}}
+        {{#> alert-body alert-body--attribute='role="alert"'}}
+          {{#> alert-title}}
+            {{#> accessibility accessibility--type="sr-only"}}
```

Comment:
You can use `{{#> screen-reader}}` instead of `{{#> accessibility}}` now. It's in the same dir as `accessibility.hbs` if you want to look at it. It's the same as `accessibility.hbs` except you don't have to pass `sr-only` to it.

Reply:
Switched to `{{#> screen-reader}}` and aligned the handlebars code more generally with the most recent updates to the Alert component.